File: ai-service/app/routes/health.py
"""Health check endpoint."""
from fastapi import APIRouter

router = APIRouter(tags=["health"])


@router.get("/health")
async def health_check():
    return {
        "status": "healthy",
        "service": "iri-ai-service",
        "version": "1.0.0",
    }

File: backend_carrier/api/health.js
const express = require('express');
const router = express.Router();

/**
 * @swagger
 * /health:
 *   get:
 *     summary: Health check endpoint
 *     description: Check if the API is running
 *     responses:
 *       200:
 *         description: API is healthy
 *         content:
 *           application/json:
 *             schema:
 *               type: object
 *               properties:
 *                 status:
 *                   type: string
 *                   example: ok
 *                 message:
 *                   type: string
 *                   example: PDF Population API is running
 */
router.get('/health', (req, res) => {
  res.json({ status: 'ok', message: 'PDF Population API is running' });
});

module.exports = router;
